code_samples/data_migration/examples/discounts/discount_code_create.yaml

```
@@ -0,0 +1,7 @@
+type: discount_code
+mode: create
+code: TEST_CODE
```

I would continue the story
```suggestion
code: summer10
```